src/Api/Handler/Request/Catalog/ProductHandler.php

```diff
@@ -108,14 +108,19 @@ public function update(
 
     /**
      * @param null|string $status
+     * @param null|array $paging
      *
      * @return \SkyHub\Api\Handler\Response\HandlerInterface
      */
-    public function products($status = null)
+    public function products($status = null, $paging = null)
```

Comment:
@Stedile93 

1. Alterar o nome da variável de `$paging` para `$pagination`;
2. Alterar a assinatura do parâmetro `$pagination` para forçar a entrada do tipo `array` já que é necessário que seja um array.